src/granite/query/assemblers/postgresql.cr

```diff
@@ -60,7 +60,7 @@ module Granite::Query::Assembler
 
     def count : Executor::Value(Model, Int64)
       where = build_where
-      order = build_order(use_default_order = false)
+      order = build_order(false)
```

Comment:
should this be `use_default_order: false`?

Reply:
It is a local variable assignment that was never used in that function anymore. So it was redundant that's why it was complaining about the variable not being used

Reply:
In term of backward compatibility, replacing it with `false` is correct.